Content.Server/Power/Components/ActivatableUIRequiresPowerComponent.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using Robust.Shared.GameObjects;
+
+namespace Content.Server.Power.Components
+{
+    [RegisterComponent]
+    public class ActivatableUIRequiresPowerComponent : Component
+    {
+        public override string Name => "ActivatableUIRequiresPower";
+    }
+}
+
```

Comment:
IMO 'ActivatableUIRequiresPower' should just be a bool datafield in ActivatableUIComponent, and not its own component.

Reply:
Someone please provide further confirmation/denial on this change, this feels like something that I might end up having to revert